### PR TITLE
Regenerate .rubocop_todo.yml

### PR DESCRIPTION
**Pull request by kakubin**
cop `RSpec/FilePath` is now obsoleted.

following message was shown when rubocop executed.

```
Error: The `RSpec/FilePath` cop has been split into `RSpec/SpecFilePathFormat` and `RSpec/SpecFilePathSuffix`.
(obsolete configuration found in .rubocop_todo.yml, please update it)
```

see also: https://github.com/rubocop/rubocop-rspec/issues/1723

### CODE REVIEW

**Comment by dblock** [comment]
Maybe use this opportunity to upgrade rubocop to a newer version and lock to it?

File: Gemfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ group :development, :test do
   gem 'rubocop'
   gem 'rubocop-capybara'
   gem 'rubocop-rake'
-  gem 'rubocop-rspec'
+  gem 'rubocop-rspec', '>= 3.0.0'
```
Comment (kakubin) [inline Gemfile:28]: https://github.com/rubocop/rubocop-rspec/releases/tag/v3.0.0

**Comment by kakubin** [comment]
@dblock 
I think the cause of downcasing header keys is in this diff. 
スクリーンショット 2024-07-29 22 49 47
Do you know anything?

**Comment by dblock** [comment]
Yes, the grape upgrade is not the intent of this PR, and it's hidden behind what you're trying to do.

You have two choices:

1. Lock version of grape and just do the rubocop upgrade here. Do a separate PR to upgrade grape.
2. Do a bundle update for everything, fix tests as needed.

**Comment by kakubin** [comment]
@dblock 
Thank you for your advice :)
Since it is a specification that different versions of grape handle a header key differently, I think [this change](https://github.com/ruby-grape/grape-swagger-rails/pull/120/commits/ad773c3d52c77b90e4e285c57a1477cb4f8c7acf) makes sense.

**Comment by padde** [comment]
@dblock probably obsoleted by #124

**Pull request closed without merging**